.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "editor.rulers": [80],
+    "editor.rulers": [120],
```

Comment:
There's support for per-filetype rulers:

https://stackoverflow.com/questions/40806241/set-vertical-rulers-in-vscode-based-on-file-type

since this is for yaml file, this deviation from the rule could have been set just for the right filetype.

The way this is setup now, the rule is applied to any filetype (and this repo also has markdown files)

Reply:
Opened a PR to correct this : https://github.com/ai-cfia/github-workflows/pull/96